README.md

```diff
@@ -163,20 +164,20 @@ node = "http://localhost:26657" # a synced node - only needed for `tx` and `broa
 
 Commands:
 
-- Generate
-- List
-- Sign
-- Broadcast
-- Raw
+- Tx - `multisig tx`
+- List - `multisig list`
+- Sign - `multisig sign`
+- Broadcast - `multisig broadcast`
+- Raw - `multisig raw`
 
-### Generate
+## Tx
 
-Generate a `unsigned.json` tx as you normally would for the given multisig address with any of the chain binaries.
-
-Then run
+The multisig tx command allows you to push transactions to S3. You can push `unsigned.json` transactions that are manually generated
+or you can use commands to generate the transactions and push it automatically to S3.
 
+### tx push
 ```
-multisig tx push --tx unsigned.json --node <node address> <chain name> <key name>
+multisig tx push <chain name> <key name>
```

Comment:
I think this still requires a `--tx` but I guess it should be changed to `--file`

Reply:
Good point, it is a required argument, however not a positional argument which is counterintuitive from a UX perspective. Issue opened to fix it.